bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/OskariDefaultStyle.js

```diff
@@ -0,0 +1,49 @@
+export const OSKARI_BLANK_STYLE = {
```

Comment:
Move this to src/react/components/StyleEditor/OskariDefaultStyle.js and use it as a base to merge the style JSON that is received from props to generate initial style.